app/assets/stylesheets/application.scss

```diff
@@ -1,11 +1,13 @@
-@import "bootstrap";
 @import "facilities";
 @import "file_uploads";
 @import "home";
 @import "sessions";
 @import "variables";
 @import "mixins";
 @import "pages/**/*";
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```

Comment:
Are these necessary with them being set in `app/javascript/packs/styles.scss` ?

Reply:
They are not. Thanks for the catch. Just removed them!